source/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -1,133 +1,137 @@
+   vda    254:0    0  8.6G  0 disk 
+   ├─vda1 254:1    0  510M  0 part 
+   ├─vda2 254:2    0   33M  0 part [SWAP]
+   └─vda3 254:3    0    8G  0 part /
 
 An example of this can also be seen in Figure 1.
 
-Checking :file:`config.json` used to build the image
-====================================================
+Method 2: Look at the image configuration YAML file 
+===================================================
 
-The second method to determine partition to check the :file:`config.json`
-file used to create prebuilt image, located in the `releases`_ repository.
-For example, to find the size of the Hyper-V\* image version number 20450,
+The second method to look at the image configuration YAML file that was
+used to produce the image. 
+
+For example, to find the size of the KVM image version number 31880,
 follow these steps:
 
 #. Go to the `releases`_ repository.
-#. Drill down into the `20450 > clear > config > image` directory.
-#. Open the :file:`hyperv-config.json` file.
-#. Locate the `PartitionLayout` key.
-
-   The example shows 512 MB for the EFI partition, 32 MB for the swap
-   partition, and 8 GB for the root partition.
+#. Drill down into the `31880 > clear > config > image` directory.
+#. Download and open the :file:`kvm.yaml` file.
+#. Locate the `targetMedia` section.
 
+   The example shows a total disk size of 8.54 GB, 512 MB for the EFI 
+   partition, 32 MB for the swap partition, and 8 GB for the root partition.
 
    .. code-block:: console
-
-      "PartitionLayout" : [ { "disk" : "hyperv.img",
-                              "partition" : 1,
-                              "size" : "512M",
-                              "type" : "EFI" },
-                              { "disk" : "hyperv.img",
-                              "partition" : 2,
-                              "size" : "32M",
-                              "type" : "swap" },
-                              { "disk" : "hyperv.img",
-                              "partition" : 3,
-                              "size" : "8G",
-                              "type" : "linux" } ],
+      :linenos:
+      :emphasize-lines: 3,9,13,18
+
+      targetMedia:
+      - name: ${bdevice}
+        size: "8.54G"
+        type: disk
+        children:
+        - name: ${bdevice}1
+          fstype: vfat
+          mountpoint: /boot
+          size: "512M"
+          type: part
+        - name: ${bdevice}2
+          fstype: swap
+          size: "32M"
+          type: part
+        - name: ${bdevice}3
+          fstype: ext4
+          mountpoint: /
+          size: "8G"
+          type: part
 
 Increase virtual disk size
 **************************
-Once you have determined the disk and partition to be increased, you are
-ready to perform the actual increase of the disk, partition, and filesystem.
 
-Power off VM and increase virtual disk size
-===========================================
+Follow these steps to increase the disk size, resize a partition, 
```

Comment:
Before increasing a virtual disk partition, increase the entire disk size of the VM. Follow these steps. 
[Here's what we discussed adding to emphasize the need to do this first.]